Zilliqa/contracts/xsgd_contract.scilla

```diff
@@ -479,15 +492,15 @@ transition Burn(value : Uint128, initiator : ByStr20, initiator_balance : Uint12
   | True =>
     new_burn_balance = builtin sub initiator_balance value;
     new_supply = builtin sub current_supply value;
+
+    (* Emits event *)
+    e = { _eventname : "Burnt"; burner : initiator; amount : value};
+    event e;
```

Comment:
For these need to be sure events don't get emitted if any errors are thrown or else will be false positive, we should have checked previously, double-check is good

Reply:
yep double checked, no events emitted before errors are thrown.